.github/actions/security-hub-scan/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: security-hub-scan
+description: Scans a zarf package with uds-security-hub
+
+inputs:
+  path:
+    description: Path to the zarf package to scan
+    required: true
+  format:
+    description: Scan output format
+    default: json
+  version:
+    description: Version of uds-security-hub to use
+    default: v0.0.11
+
+outputs:
+  cve-scan:
+    description: A filename that includes the results in input.format
+    value: ${{ steps.scan.outputs.cve-scan }}
+  cosign-bundle:
+    description: A file that includes the cosign bundle for verifying authenticity
+    value: ${{ steps.cosign.outputs.bundle }}
+
+runs:
+  using: composite
+  steps:
+    - name: install cosign
+      uses: sigstore/cosign-installer@4959ce089c160fddf62f7b42464195ba1a56d382 # v3.6.0
+
+    - name: create tmpdir for output
+      id: tmpdir
+      run: |
+        tmpdir=$(mktemp -d)
+        echo "dir=$tmpdir" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - name: use uds-security-hub to scan a package
+      id: scan
+      run: |
+        OUTPUT_FILE=${{ steps.tmpdir.outputs.dir }}/cve-scan.${{ inputs.format }}
+
+        docker run --rm \
```

Comment:
I'm not opposed to this, but are there any references to why this is a script in an action calling a docker container rather than a uds task. Also why this functionality couldnt be vendored into uds-cli so it would just be `uds scan <thing to scan>` to execute?

Reply:
This is an agreed upon interim solution, but I do think what you mention makes a lot of sense long term.

Reply:
Linking this to the most relevant issue I think is there to give folks breadcrumbs: https://github.com/defenseunicorns/uds-security-hub/issues/176 cc @koesbong as well if there is more context to give (I loosely followed some stuff but only have a slice myself)

Reply:
is there anything I should do here? Should I take a stab at making this work with the tasks system, even though it's currently going to be running with the docker command?

Reply:
Yeah I think it still probably makes sense to make this a task and call it in the action or from workflow. We do the same where it's wrapped in a docker run for our playwright tests. 

That way someone can run locally via `uds run` as well.

Reply:
@zachariahmiller I took a stab at writing this as a uds task. I left the cosign bit out because it requires cosign to be installed, so the cosign portion is only in the action at the moment. Can you take another look when you get a chance?